### PR TITLE
Mq -> MQ

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/Serilog.Sinks.RabbitMQ/Sinks/RabbitMQ/RabbitMQClient.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@
 namespace Serilog.Sinks.RabbitMQ;
 
 /// <summary>
-/// RabbitMqClient - this class is the engine that lets you send messages to RabbitMq.
+/// RabbitMqClient - this class is the engine that lets you send messages to RabbitMQ.
```
Comment (ArieGato) [inline src/Serilog.Sinks.RabbitMQ/Sinks/RabbitMQ/RabbitMQClient.cs:21]:
```suggestion
/// RabbitMQClient - this class is the engine that lets you send messages to RabbitMQ.
```

**Review by ArieGato — APPROVED** [review]

File: src/Serilog.Sinks.RabbitMQ/LoggerConfigurationRabbitMQExtension.cs [hunk 2]
```diff
@@ -25,9 +25,9 @@
 namespace Serilog;
 
 /// <summary>
-/// Extension method to configure Serilog with a Sink for RabbitMq.
+/// Extension method to configure Serilog with a Sink for RabbitMQ.
 /// </summary>
-public static class LoggerConfigurationRabbitMqExtension
+public static class LoggerConfigurationRabbitMQExtension
```
Comment (ArieGato) [inline src/Serilog.Sinks.RabbitMQ/LoggerConfigurationRabbitMQExtension.cs:30]: This is potentially a breaking change. Should this be mentioned in the Breaking changes section of the Readme?

**Review by ArieGato — CHANGES_REQUESTED** [review]

File: src/Serilog.Sinks.RabbitMQ/LoggerConfigurationRabbitMQExtension.cs (hunk 2 above)
Comment (sungam3r) [reply src/Serilog.Sinks.RabbitMQ/LoggerConfigurationRabbitMQExtension.cs:30]: done

**Review by ArieGato — APPROVED** [review]
Looks good to me

**Pull request merged**